src/main/java/fi/aalto/cs/apluscourses/presentation/exercise/EmptyExercisesTreeViewModel.java

```diff
@@ -0,0 +1,23 @@
+package fi.aalto.cs.apluscourses.presentation.exercise;
+
+import fi.aalto.cs.apluscourses.model.ExerciseGroup;
+import fi.aalto.cs.apluscourses.presentation.filter.Options;
+import java.util.List;
+import org.jetbrains.annotations.NotNull;
+
+
+public class EmptyExercisesTreeViewModel extends ExercisesTreeViewModel {
+
+  /**
+   * Construct an exercises tree view model from the given exercise groups.
+   */
+  public EmptyExercisesTreeViewModel(@NotNull List<ExerciseGroup> exerciseGroups,
+                                     @NotNull Options filterOptions) {
+    super(exerciseGroups, filterOptions);
+  }
```

Comment:
Since it's supposed to be empty, I would suggest that this constructor doesn't take a `List` parameter at all, and instead it always passes an empty list to the superclass constructor